pkg/cmd/workload/run.go

```diff
@@ -332,6 +333,14 @@ func runRun(gen workload.Generator, urls []string, dbName string) error {
 					_ = jsonEnc.Encode(t.Snapshot())
 				}
 			})
+		// Once the load generator is fully ramped up, we reset the histogram and the start time, to
```

Comment:
It might be nice to also print something to help visually mark where the ramp ended. Perhaps that could just be resetting `i` to 0 so that the header gets reprinted, or a short message of some sort.

Not a big deal though, your call.